components/using_components.rst

```diff
@@ -40,7 +40,7 @@ If you know you need a specific version of the library, add that to the command:
 
 .. code-block:: bash
 
-    $ composer require symfony/finder
+    $ composer require symfony/finder 2.6.2
```

Comment:
I agree about removing it. This page is not about documenting all features of ``composer require``, and specifying an exact match requirement is a mistake most of the time (you would want at least ``2.6.*`` to allow getting bug fixes and security patches when running a ``composer update``), so we should not show such example IMO